dataplane/ebpf/Cargo.toml

```diff
@@ -8,7 +8,8 @@ publish = false
 aya-bpf = { git = "https://github.com/aya-rs/aya", branch = "main" }
 aya-log-ebpf = { git = "https://github.com/aya-rs/aya", branch = "main" }
 common = { path = "../common" }
-memoffset = "0.8"
+memoffset = "0.9"
+network-types = { git = "https://github.com/vadorovsky/network-types", rev = "fd4ecb4" }
```

Comment:
You should just be able to use the released version here right? 
`network-types = "0.0.4"`

https://crates.io/crates/network-types

Reply:
Actually, I made a new release just now. I think you should be able to switch to it. https://crates.io/crates/network-types/0.0.5

Reply:
See: https://github.com/kubernetes-sigs/blixt/pull/147#discussion_r1409305695

tldr; we're pinning a SHA (as there's some stuff in `main` we need)

Reply:
Oh cool, and now @vadorovsky dropped a new release anyway (thanks!):

https://github.com/kubernetes-sigs/blixt/pull/147#pullrequestreview-1757728964

So we can just use that. :tada: